src/main/scala/iog/psg/cardano/CardanoApiImpl.scala

```diff
@@ -27,48 +28,33 @@ private class CardanoApiImpl(baseUriWithPort: String)(implicit ec: ExecutionCont
   implicit val config: Configuration = Configuration.default.withSnakeCaseMemberNames
 
   /**
-   * @inheritdoc
-   */
-  override def listWallets: CardanoApiRequest[Seq[Wallet]] = CardanoApiRequest(
-    HttpRequest(
-      uri = wallets,
-      method = GET
-    ),
-    _.toWallets
-  )
+    * @inheritdoc
+    */
+  override def listWallets: CardanoApiRequest[Seq[Wallet]] =
+    CardanoApiRequest(HttpRequest(uri = wallets, method = GET), _.toWallets)
```

Comment:
This is what I'd like to avoid (without turning off formatter) It would be good to keep the original format